docs/source/commandline/commandline-all.rst

```diff
@@ -1708,18 +2719,17 @@ Core Video OpenGL GLSL Options
 
 **-glsl_shader_mame9**
 
+    Set a custom OpenGL GLSL shader effect to the internal systcm screen in the
+    given slot. MAME does not include a vast selection of shaders by default;
+    more can be found online.
 
-    Custom OpenGL GLSL shader set MAME bitmap in the provided slot (0-9); one
-    can be applied to each slot.
-
-    [todo: better details on usage at some point.
-    See http://forums.bannister.org/ubbthreads.php?ubb=showflat&Number=100988#Post100988 ]
-
+    Example:
+        .. code-block:: bash
 
+            mame64 suprmrio -gl_glsl -glsl_shader_mame0 NTSC\NTSC_chain -glsl_shader_mame1 CRT-geom\CRT-geom
```

Comment:
Please use `/` as a directory separator in examples – it’s less likely to be problematic.

Reply:
Most of our users are going to be Windows; macOS and Linux users would already know this stuff-- but I'll change it.

Reply:
Note that those plugins also don't come with MAME. We only ship a few very basic plugins.